bin/run-file-clock.sh

```diff
@@ -4,7 +4,12 @@
 # License, v. 2.0. If a copy of the MPL was not distributed with this
 # file, You can obtain one at https://mozilla.org/MPL/2.0/.
 
-bin/run-db-download.py --force
+# On demos, we don't want to download the latest DB, else we'll lose CMS data
```

Comment:
This only runs once at startup. Do we not even want it to happen then?

Reply:
No because it then replaces the DB that's baked into the image, which contains the SSO users added during the build step. It was the only way I could find to run that management command in this Cloud Build + Cloud Run setup.

(Note that the image containing the SSO user info is stored in a private container repository, not a public one, of course)

Does that sound OK/viable to you?

Reply:
Also, "startup" in Cloud Run terms can be after it goes idle after 15 mins (unless we specify always-on mode), so if somoene sets a Switch via a migration (our current hack), this forced download will overwrite those changes - which I think is what Stephanie encountered on Friday

Reply:
Ah, so we'd have to run the user add thing after this step during pod start. That could work too, but we can go ahead and get this out.